queries/prisma/highlights.scm

```diff
@@ -0,0 +1,26 @@
+[
+ "datasource"
+ "enum"
+ "generator"
+ "model"
+] @keyword
+
+(comment) @comment
+(developer_comment) @comment
+
+(arguments) @property
+(attribute) @function
+(call_expression) @function
+(column_type) @type
+(enumeral) @constant
+(identifier) @variable
+(string) @string
+
+"(" @punctuation.bracket
+")" @punctuation.bracket
+"[" @punctuation.bracket
+"]" @punctuation.bracket
+"{" @punctuation.bracket
+"}" @punctuation.bracket
+"=" @operator
```

Comment:
Could you use `[ ...... ] @punctuation.bracket` here to summarize the different brackets in one alteration